ocaml/lib/pickles_bindings.ml

```diff
@@ -461,13 +463,26 @@ let pickles_compile (choices : pickles_rule_js array)
   (* translate method circuits from JS *)
   let public_input_size = signature##.publicInputSize in
   let public_output_size = signature##.publicOutputSize in
+  let override_wrap_domain =
+    match Js.Optdef.to_option signature##.overrideWrapDomain with
```

Comment:
Berkeley uses
```
  let override_wrap_domain =
    Js.Optdef.to_option signature##.overrideWrapDomain
    |> Option.map ~f:Pickles_base.Proofs_verified.of_int
  in
```

Reply:
updated!

Reply:
I make a commit changing to it, feel free to remove it if you do not agree with

Reply:
Too late :grimacing:. You're a more morning person than me :)

Reply:
I already made that commit :D